src/components/VSelect/index.js

```diff
@@ -44,9 +47,11 @@ const wrapper = {
       deprecate('<v-select editable>', '<v-overflow-btn editable>', wrapper, parent)
     }
 
-    if (props.autocomplete || props.combobox || props.tags) {
-      data.attrs.combobox = props.combobox
-      data.attrs.tags = props.tags
+    if (props.combobox || props.tags) {
+      data.attrs.multiple = props.tags
```

Comment:
`multiple` must always be passed though, otherwise `<v-select multiple>` won't work